src/external/mint.rs

```diff
@@ -18,6 +18,8 @@ use crate::bill::{
 };
 use crate::web::RequestToMintBitcreditBillForm;
 
+// Usage of tokio::main to spawn a new runtime is necessary here, because Wallet is'nt Send - but
+// this logic will be replaced soon
```

Comment:
Yes, not optimal to have multiple runtimes. Client should become a ```Arc<Mutex<Wallet>>>``` or even better ```dyn WalletApi```.

Reply:
Yes, I thought that would be the nicest solution as well, but since all of the mint-related stuff will quite soon be entirely replaced, the effort doesn't seem to be worth it. :+1: